.storybook/main.ts

```diff
@@ -1,26 +1,17 @@
+import type { StorybookConfig } from "@storybook/react-webpack5";
 import path from "path";
 import remarkGfm from "remark-gfm";
 
-module.exports = {
-    stories: [
-        "../stories/**/!(*.stories).mdx",
-        "../stories/**/*.stories.@(ts|tsx)",
-    ],
+const config: StorybookConfig = {
+    stories: ["../stories/**/*.mdx", "../stories/**/*.stories.@(ts|tsx)"],
     addons: [
+        "@storybook/addon-webpack5-compiler-swc",
```

Comment:
what does this addon do?

Reply:
with a fresh init of storybook, this was the compiler set up. can't rmb which option this was but there were 2 suggested..